tests/cds-rdm/test_full_migration.py

```diff
@@ -33,7 +33,7 @@
 def suite_multi_field(record):
     """2684743."""
     dict_rec = record.to_dict()
-    assert dict_rec["created"] == "2019-07-29T00:00:00+00:00"
+    assert dict_rec["created"] == "2019-08-02T22:21:26+00:00"
```

Comment:
it seems the test was correct? it 
could you add another test case were we check if new version has correct (new) creation date?

Reply:
Right, since the first version is using the record date, it's correct as it was